tests/testthat/testImportSNV.TCGA.R

```diff
@@ -0,0 +1,181 @@
+
+    # TEST fMAF, rSNV, silent parameters ============================================
+    # run the module on fMAF and rSNV and silent as parameters
+    moduleOutPre2 <- importSNV.TCGA(c("testInPre"), "module02", silent=TRUE)
+    modulePre2 <- read.csv(moduleOutPre2, sep = "\t")
+    # add to prexisting rSNV file
+    all.equal(trueOutPre, modulePre2)
+    #===============================================================================
+
+    # TEST fMAF, rSNV, silent, wrtieLog parameters =================================
+    # run the module on fMAF, rSNV, silent and writeLog as parameters
+    moduleOutPre3 <- importSNV.TCGA(c("testInPre"), "module02", silent=FALSE, writeLog=TRUE)
+    modulePre3 <- read.csv(moduleOutPre3, sep = "\t")
+    # add to prexisting rSNV file
+    all.equal(trueOutPre, modulePre3)
+    #===============================================================================
+
+
+    # test input with rSNV as input and NA values -------------------------------------------
+    # have an input file, rows too big to import manually here (testInNA)
+    input0 <- c("Hugo_Symbol" , "Entrez_Gene_Id" , "Center" , "NCBI_Build" , "Chromosome" , "Start_Position" , "End_Position" , "Strand" , "Variant_Classification" , "Variant_Type" , "Reference_Allele" , "Tumor_Seq_Allele1" , "Tumor_Seq_Allele2" , "dbSNP_RS" , "dbSNP_Val_Status" , "Tumor_Sample_Barcode" , "Matched_Norm_Sample_Barcode" , "Match_Norm_Seq_Allele1" , "Match_Norm_Seq_Allele2" , "Tumor_Validation_Allele1" , "Tumor_Validation_Allele2" , "Match_Norm_Validation_Allele1" , "Match_Norm_Validation_Allele2" , "Verification_Status" , "Validation_Status" , "Mutation_Status" )
+    input3 <- c("VWA5B1" , 127731 , "WUGSC" , "GRCh38" , "chr1" , 20319393 , 20319393 , "+" , "Missense_Mutation" , "SNP" , "C" , "C" , "T" , "novel" , "TCGA-QQ-A8VF-01A-11D-A37C-09", "TCGA-QQ-A8VF-10A-01D-A37F-09" , "Somatic" , "Illumina" , "HiSeq" , "2000" , "2109690d-e8b1-4b5c-9a39-21af273324cf" , "fe2daf93-56f1-4236-9b58-bf18f0e47d3a" , "c.853C>T" , "p.Pro285Ser" , "p.P285S" , "ENST00000375079")
+
+    trueInNA <- rbind(input0,input3)
+
+    write.table(trueInNA,  file="testInNA", sep="\t", append=FALSE, row.names =
+                    FALSE, col.names = FALSE, quote = FALSE)
+
+    # test output for rSNV as input and NA values
+    header <- c("sym", "chr", "start", "end", "strand", "class", "type", "aRef", "a1", "a2", "TCGA", "UUID")
+
+    trueOutNA <- rbind(header)
+
+    # TEST fMAF, rSNV, silent, wrtieLog parameters, na.rm ==========================
+    # run the module on fMAF, rSNV, silent, writeLog and na.rm as parameters
+    moduleOutNA <- importSNV.TCGA(c("testInNA"), silent=FALSE, writeLog=FALSE, na.rm=TRUE)
+    moduleNA <- read.csv(moduleOutNA, sep = "\t")
+    # add to prexisting rSNV file
+    all.equal(trueOutNA, moduleNA)
+    #===============================================================================
+})
+
+
+test_that("a corrupt input does not lead to corrupted output", {
```

Comment:
I think a check for an incomplete last line would be great (no carriage return). That's the hallmark of a truncated txt file. Probably whatever function you use to read the file in the first place throws an error or warning anyway that you can test for.